[bitnami/sealed-secrets] panic: name contains illegal character "

### Name and Version

bitnami/sealed-secrets 1.6.0

### What architecture are you using?

arm64 and amd64 (mixed heterogeneous)

### What steps will reproduce the bug?

1. Update helm chart [from 1.5.15 to 1.6.0](https://gitlab.com/deepcypher/dc-kc/-/commit/823d65e5d19ce6e8ee40322518d8f9f646b5b5c5)
2. Notice that new sealed-secrets pods start to fail using the same values.
3. get the logs to find: panic: name contains illegal character "

### Are you using any custom parameters or values?
```yaml
sealed-secrets:
  ingress:
    enabled: false
    hostname: secrets.deepcypher.me
  createController: true
  networkPolicy:
    enabled: true
```

### What is the expected behaviour?

Pods would rolling update to 1.6.0 in a healthy state.

### What do you see instead?

![old-to-new](https://github.com/bitnami/charts/assets/10534713/c80d9184-8e09-422b-93b6-f98daa8514f5)

New (failing) pod logs (1.6.0) [at line 96](https://github.com/bitnami-labs/sealed-secrets/blob/7b789fddacf62fc0254548d61f6b60e64a02c5fa/cmd/controller/main.go#L96):
```
controller version: v0.24.4
2023/12/02 08:33:06 Starting sealed-secrets controller version: v0.24.4
panic: name contains illegal character "
goroutine 1 [running]:
main.mainE({0x175a3e0, 0x40001a6008}, 0x400005e768?, 0x4884c?, {0x40001aa040, 0x4, 0x4})
	/bitnami/blacksmith-sandox/sealed-secrets-0.24.4/src/github.com/bitnami-labs/sealed-secrets/cmd/controller/main.go:97 +0x1fc
main.main()
	/bitnami/blacksmith-sandox/sealed-secrets-0.24.4/src/github.com/bitnami-labs/sealed-secrets/cmd/controller/main.go:103 +0x54
```

Old (working) pod logs (1.5.15):
```
controller version: v0.24.4
2023/11/29 20:53:10 Starting sealed-secrets controller version: v0.24.4
Searching for existing private keys
----- sealed-secrets-keybm7z5
----- sealed-secrets-keymprrf
----- sealed-secrets-keyj4nbb
----- sealed-secrets-key72gmr
HTTP server serving on :8080
Updating cloudflared/tunnel-credentials
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"cloudflared", Name:"tunnel-credentials", UID:"32d7a3aa-8050-4fc1-b7da-a5190b39f86a", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"48799394", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
Updating renovate/renovate-gh
update suppressed, no changes in sealed secret spec of cloudflared/tunnel-credentials
update suppressed, no changes in sealed secret spec of renovate/renovate-gh
Updating renovate/renovate-gl
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"renovate", Name:"renovate-gh", UID:"0b4dcbfd-1b32-4624-b46f-00f64ec0bc1e", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"43314819", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
Updating renovate/renovate-gt
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"renovate", Name:"renovate-gl", UID:"4101a157-12eb-474f-8aac-cbf45b91857e", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"43314825", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
update suppressed, no changes in sealed secret spec of renovate/renovate-gl
Updating velero/cloud-credentials
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"renovate", Name:"renovate-gt", UID:"98b3edfb-7865-494a-a0ad-ce8986c02368", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"43314832", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"velero", Name:"cloud-credentials", UID:"32a8247f-0845-43f5-86f2-f2d7a74d5c88", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"48013093", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
update suppressed, no changes in sealed secret spec of velero/cloud-credentials
update suppressed, no changes in sealed secret spec of renovate/renovate-gt
update suppressed, no changes in sealed secret spec of velero/cloud-credentials
update suppressed, no changes in sealed secret spec of cloudflared/tunnel-credentials
Updating cloudflared/tunnel-token
Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"cloudflared", Name:"tunnel-token", UID:"9a46e343-5a18-458d-8f5b-cfdbe6a445dc", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"49337264", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
update suppressed, no changes in sealed secret spec of cloudflared/tunnel-token
update suppressed, no changes in sealed secret spec of cloudflared/tunnel-token
update suppressed, no changes in sealed secret spec of cloudflared/tunnel-token
```

### Additional information

A full history and setup of this issue in the last two days is available here: https://gitlab.com/deepcypher/dc-kc/-/tree/823d65e5d19ce6e8ee40322518d8f9f646b5b5c5/charts/sealed-secrets

--- a/bitnami/sealed-secrets/Chart.yaml
+++ b/bitnami/sealed-secrets/Chart.yaml
@@ -29,4 +29,4 @@ name: sealed-secrets
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/sealed-secrets
 - https://github.com/bitnami-labs/sealed-secrets
-version: 1.6.0
+version: 1.6.1

--- a/bitnami/sealed-secrets/templates/deployment.yaml
+++ b/bitnami/sealed-secrets/templates/deployment.yaml
@@ -89,7 +89,7 @@ spec:
           {{- else }}
           args:
             - "--key-prefix"
-            - {{ printf "%s" (include "sealed-secrets.secretName" .) | quote }}
+            - {{ printf "%s" (include "sealed-secrets.secretName" .) }}
             {{- if .Values.updateStatus }}
             - "--update-status"
             {{- end }}

